src/input_output/FGXMLElement.cpp

```diff
@@ -165,8 +168,11 @@ Element::Element(const string& nm)
     // Mass flow
     convert["KG/MIN"]["LBS/MIN"] = convert["KG"]["LBS"];
     convert["KG/SEC"]["LBS/SEC"] = convert["KG"]["LBS"];
-    convert ["N/SEC"]["LBS/SEC"] = 0.224808943;
-    convert ["LBS/SEC"]["N/SEC"] = 1.0/convert ["N/SEC"]["LBS/SEC"];
+    convert["N/SEC"]["LBS/SEC"] = 0.224808943;
+    convert["LBS/SEC"]["N/SEC"] = 1.0/convert ["N/SEC"]["LBS/SEC"];
```

Comment:
For my piece of mind 😅 
```suggestion
    convert["LBS/SEC"]["N/SEC"] = 1.0/convert["N/SEC"]["LBS/SEC"];
```